tests/run_smoke_tests.sh

```diff
@@ -25,4 +25,4 @@ else
     test_spec=tests/test_smoke.py::"${test}"
 fi
 
-pytest -s -n 16 -q --tb=short --disable-warnings "$test_spec"
```

Comment:
We don't have the `run_smoke_tests.sh` in our current master branch now. We can add this new option to `pyproject.toml`.

Reply:
Fixed.